src/api/repos/user-repo.ts

```diff
@@ -66,6 +66,25 @@ const getUserFavoriteActivityList = async (id: string) => {
     };
 };
 
+const checkActivityFavoritedByUser = async (id: string, activity: string) => {
```

Comment:
call it `activityId` instead of `activity`. and if you follow the comment below from this same file, change from `string` to `number` in here and `user-schema`